daemon/rpc_test.go

```diff
@@ -69,12 +69,20 @@ var (
 )
 
 func testRPC() *RPC {
-	api := core.NewDefaultAPI(
-		"1.0.0",
-		"",
-		http.DefaultClient,
-		response.NoopValidator{},
+	type mockLoginTokenManager struct {
```

Comment:
This is also done in `auth/token_test.go`, maybe it could be moved to somewhere in `test/mock` and reused?